src/types/FamilyUnitAgesProps.ts

```diff
@@ -0,0 +1,5 @@
+export default interface FamilyUnitAgesProps {
+    under3: number,
```

Comment:
Be careful here. Does `under3` mean age <= 3? You are excluding 16 also, since `between3and15` does not include 16 and `over16` does not include it either.